components/log-viewer-webui/server/src/plugins/MongoReplicaServerPlugin.ts

```diff
@@ -0,0 +1,272 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
Let's remove this and move the classes into separate files then import as needed